IdentityCore/src/configuration/MSIDConfiguration.m

```diff
@@ -53,19 +62,36 @@ - (instancetype)initWithAuthority:(MSIDAuthority *)authority
         _redirectUri = redirectUri;
         _clientId = clientId;
         _target = target;
```

Comment:
Should we remove `target` ivar and calculate it in real time? Right now we can assign a new value to scopes/resource after the `target` was initialized (because properties are writable), so the target would be out of sync with scopes/resource.

Reply:
Scopes and resource are readonly in the public interface (.h).

Reply:
you are right, but `target` is still writable. If I set new value to it, scopes/resources would not be updated.

Reply:
you're right, let me make it readonly too, I don't see a reason why it needs to be writable.

Reply:
Addressed